modules/op-mitigating-extra-pipeline-resource-consumption.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// */openshift_pipelines/uninstalling-pipelines.adoc
+
+[id='op-mitigating-extra-pipeline-resource-consumption_{context}']
+= Mitigating extra resource consumption in pipelines
+
+When you have resource limits set on the container of your pod, {product-title} sums up the resource limits requested as all containers run at the same time.
+
+To consume the bare minimum amount of resources needed to execute one step at a time from the invoked task, {pipelines-title} requests the maximum CPU, memory, and ephemeral storage as specified in the step that requires the most amount of resources. This ensures that the resource requirements of the step that needs the most amount of resources as well as the steps that need a lesser amount of resources are met. Requests other than the maximum values are set to zero.
+
+However, this behavior can lead to higher resource usage than required. If you use resource quotas, this could also lead to unschedulable pods.
+
+For example, in a task that has two steps that uses scripts and that does not define any resource limits and requests.
```

Comment:
This example sounds incomplete. Maybe you should include the next sentence in the same paragraph.